Resources/Prototypes/Entities/Mobs/Species/arachnid.yml

```diff
@@ -30,8 +33,6 @@
     spawned:
       - id: FoodMeatSpider
         amount: 5
-  - type: Inventory
-    templateId: arachnid
```

Comment:
I would say "consider commenting this out instead of removing it" but it looks like these two lines were moved elsewhere. Instead of adding lines elsewhere, maybe restore this part and just add the ` speciesId: arachnid # DeltaV - allow arachnid clothing sprite variants` line.

Reply:
Alright, I moved the `type: Inventory` lines back where they originally were and added the `speciesID` line underneath them :).